ci/pipelines/deploy-to-test.yml

```diff
@@ -2787,6 +2795,91 @@ jobs:
           image: cardid-ecr-registry-test/image.tar
           additional_tags: cardid-ecr-registry-test/tag
           
+  - name: build-webhooks
+    plan:
+      - get: webhooks-git-release
+        trigger: true
+      - get: pay-ci
+      - task: parse-release-tag
+        file: pay-ci/ci/tasks/parse-release-tag.yml
+        input_mapping:
+          git-release: webhooks-git-release
+      - task: build-jar
+        config:
+          container_limits: {}
+          platform: linux
+          image_resource:
+            type: registry-image
+            source:
+              repository: maven
+              tag: 3-openjdk-17
+          inputs:
+            - name: webhooks-git-release
+          outputs:
+            - name: jar
+          run:
+            path: bash
+            args:
+            - -ec
+            - |
+              ls -lrt
+              pwd
+
+              cd webhooks-git-release
+              mvn clean package
+              cp -r . ../jar
+
+      - task: build-image
+        privileged: true
+        config:
+          container_limits: {}
+          platform: linux
+          image_resource:
+            type: registry-image
+            source:
+              repository: govukpay/concourse-runner
+              tag: latest
+          inputs:
+            - name: jar
+          outputs:
+            - name: local_image
+          params:
+            app_name: webhooks
+          run:
+            path: bash
```

Comment:
Guessing this section is based on the `build-docker-image.yml` task, with a couple of tweaks? Could we reuse that task going forward?